_posts/2018-07-31-bazel-0.16.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: posts
+title: "Bazel 0.16"
+authors:
+  - cparsons
```

Comment:
I think you need to be in this list too:
https://github.com/bazelbuild/bazel-blog/blob/master/_config.yml

Reply:
Requested in https://github.com/bazelbuild/bazel-blog/pull/109